src/components/bookStyles.css

```diff
@@ -0,0 +1,35 @@
+.book-item {
```

Comment:
Sounds a good idea, but we are using tailwindcss so no css is used

You can delete this file and update classnames